docs/internal_api.yml

```diff
@@ -228,7 +228,7 @@ paths:
       summary: Upload mender artifact
       description: |
         Upload mender artifact to a specific tenant. Multipart request with meta and artifact.
-        Supports artifact (versions v1, v2, and v3)[https://docs.mender.io/development/architecture/mender-artifacts#versions].
+        Supports artifact [versions v1, v2, 3](https://docs.mender.io/overview/artifact#versions).
```

Comment:
What happened to the `v` in `v3`?

Reply:
Ups. I just copied if from `management_api.yml`. I'll fix in both places.